Dockerfile

```diff
@@ -8,7 +8,8 @@ WORKDIR /directus
 
 ENV NODE_OPTIONS=--max-old-space-size=8192
 
-RUN corepack enable && corepack prepare pnpm@8.0.0 --activate
+COPY package.json .
+RUN corepack enable && corepack prepare
```

Comment:
This means the `RUN corepack ...` statement is rerun (not from cache) each time the `package.json` file changes, but in my opinion, this is neglectable since the
-  the installation is very fast
    ```console
    => [builder 3/9] COPY package.json .                                        0.0s
    => [builder 4/9] RUN corepack enable && corepack prepare   1.2s
    ```
- the `package.json` file doesn't change that much and if it does, the lockfile mostly changes as well